src/main/controllers/features/unincorporated/unincorporatedConfirmYourBusinessAddressController.ts

```diff
@@ -0,0 +1,28 @@
+import { Session } from "@companieshouse/node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { BUSINESS_ADDRESS, USER_DATA } from "../../../common/__utils/constants";
+import * as config from "../../../config";
+import { BASE_URL, UNINCORPORATED_BUSINESS_ADDRESS_CONFIRM, UNINCORPORATED_BUSINESS_ADDRESS_LOOKUP, UNINCORPORATED_BUSINESS_ADDRESS_MANUAL_ENTRY, UNINCORPORATED_WHAT_IS_THE_CORRESPONDENCE_ADDRESS } from "../../../types/pageURL";
+import { addLangToUrl, getLocaleInfo, getLocalesService, selectLang } from "../../../utils/localise";
+import { ACSPData } from "main/model/ACSPData";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const acspData: ACSPData = session?.getExtraData(USER_DATA)!;
+    const businessAddress = session?.getExtraData(BUSINESS_ADDRESS);
```

Comment:
This should not be used now.. Aim to get values from User_data not on their own

Reply:
We do not have a businessAddress in ACSPData at the moment do you want me to add that or I can use addresses ?